src/3d/qgs3daxis.h

```diff
@@ -84,6 +81,11 @@ class _3D_EXPORT Qgs3DAxis : public QObject
      */
     void onViewportScaleFactorChanged( double scaleFactor );
 
+    /**
+     * Returns TRUE if the 3D axis controller will handle the specified \a event,
+     */
+    bool willHandleEvent( QEvent *event );
```

Comment:
Why is this called "will" handle event, when it actually DOES handle the event?

I think it should be renamed to just handleEvent, and the documentation updated to reflect this

Reply:
it is copied from QgsCameraController, I thought it was some kind of QGIS design. Fixed!